t/extutils-makemaker-module-with-manifest.skip.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+PATH="$PATH:../../../bin"
+
+setup() {
+  cd t/test-data/extutils-makemaker-module-with-manifest.skip.bats
+}
```

Comment:
Does `setup()` ever get run?

Reply:
no idea, I just copied the `t/module_build_module.bats` file :thinking:

Reply:
and as I can see it is the same in `t/dzil_module.bats` as well

Reply:
but now I can see I need to include the test files in the GA yaml file as well.

Reply:
and it seems the build.pl test is also missing from it.